x-pack/plugin/inference/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/xpack/application/CohereServiceUpgradeIT.java

```diff
@@ -71,7 +71,7 @@ public void testCohereEmbeddings() throws IOException {
             cohereEmbeddingsServer.enqueue(new MockResponse().setResponseCode(200).setBody(embeddingResponseFloat()));
             put(oldClusterIdFloat, embeddingConfigFloat(getUrl(cohereEmbeddingsServer)), TaskType.TEXT_EMBEDDING);
 
-            var configs = (List<Map<String, Object>>) get(TaskType.TEXT_EMBEDDING, oldClusterIdInt8).get("models");
+            var configs = (List<Map<String, Object>>) get(TaskType.TEXT_EMBEDDING, oldClusterIdInt8).get("endpoints");
```

Comment:
Are these changes trying to address the fact that these tests are muted? Or is it to bring the naming in alignment?

https://github.com/elastic/elasticsearch/issues/107887

Reply:
This change won't fix these tests, but this change is required for the tests to be fixed/unmuted. I just noticed that these tests were broken when I copied them into my new mixed cluster tests.